R/simulation-batch.R

```diff
@@ -110,6 +111,10 @@ SimulationBatch <- R6::R6Class(
       } else {
         private$throwPropertyIsReadonly("runValuesIds")
       }
+    },
+    #' @field id The id of the .NET wrapped object. (read-only)
```

Comment:
```suggestion
    #' @field id A unique identifier for the `.NET` wrapped object. (read-only)
```

Reply:
it sounds like it's not the one from .net where in fact, it IS

Reply:
I would leave it as "ID" as this is how the field in .NET is called..